src/components/modules/contacts/hooks/actions-contact.hook.ts

```
@@ -0,0 +1,136 @@
+      lastName: "",
+      email: "",
+      address: "",
+      category: ["personal"],
+    },
+    mode: "onChange",
+  });
+
+  useEffect(() => {
+    if (formData) {
+      Object.entries(formData).forEach(([key, value]) => {
+        form.setValue(key as keyof z.infer<typeof formSchema>, value as any);
```

ESLint

[error] Unexpected any. Specify a different type.

(@typescript-eslint/no-explicit-any)